src/Avalonia.Animation/AnimationInstance`1.cs

```diff
@@ -188,10 +188,7 @@ private void InternalStep(TimeSpan systemTime)
 
             if (!_isLooping)
             {
-                if (_currentIteration > _repeatCount)
-                    DoComplete();
-
-                if (time > iterationEndpoint)
+                if ((_currentIteration > _repeatCount) | (time > iterationEndpoint))
```

Comment:
WARN: byte operator `|` instead of bool operator `||`.

bool operator is preferred as it can skip the right side evaluation when the left side is `true`.

Reply:
No i deliberately set it to eager evaluation since that code needs to check the right-side condition

Reply:
If there's no side-effects in either operand then there's no reason not to use lazy evaluation.

Reply:
@jkoritzinsky right, i forgot about that. I just wanted to be doubly-sure that the conditions will be evaluated

Reply:
They will both be evaluated. Can you either switch it to use `||` or add a comment on why it needs to be `|`?

Reply:
I'll just change it back to `||` since my justification for not using that is already moot